src/renderer/data/accounts/getAccount.ts

```diff
@@ -94,8 +94,8 @@ export const renderData = (account: AccountInfo | undefined) => {
 };
 
 // TODO: this should look up a persistent key: string map
-export const getHumanName = (key: AccountInfo | sol.PublicKey) => {
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const getHumanName = (_key: AccountInfo | sol.PublicKey) => {
   /* eslint-disable no-console */
```

Comment:
OI ! this one too :p

Reply:
this one what. this isn't really related to the pr tbh but it was spamming the console something terrible

Reply:
yup - its just that in removing the console.log, you can thus also remove the eslint-disable :)

we need a lint instruction linter-er